test/contract/ffc-demo-payment-service.test.js

```diff
@@ -0,0 +1,44 @@
+describe('Pact Verification', () => {
+  const { Verifier } = require('@pact-foundation/pact')
+  const mockScheduleRepository = require('../unit/server/repository/schedule-repository.mock')
+  const path = require('path')
+
+  let createServer
+  let server
+
+  beforeAll(async () => {
+    const oktaJwtVerifier = require('../../server/plugins/auth-okta/okta-jwt-verifier')
+    jest.mock('../../server/plugins/auth-okta/okta-jwt-verifier')
+    oktaJwtVerifier.verifyAccessToken.mockImplementation(() => Promise.resolve({ claims: { roles: ['payment-admin'] } }))
+
+    jest.mock('../../server/repository/schedule-repository', () => mockScheduleRepository)
+    createServer = require('../../server')
+  })
+
+  beforeEach(async () => {
+    server = await createServer()
+    await server.start()
+  })
+
+  test('validates the expectations of ffc-demo-payment-web', async (done) => {
+    const opts = {
+      providerBaseUrl: 'http://localhost:3004',
```

Comment:
Might be a good idea to source the port from the `config`?  Then if it changes we don't need to update the hard coded value in the test.

Reply:
Resolved by push a696b12